modules/gapi/src/api/render_ocv.hpp

```diff
@@ -0,0 +1,30 @@
+#include <vector>
+#include "render_priv.hpp"
+
+#ifndef OPENCV_RENDER_OCV_HPP
+#define OPENCV_RENDER_OCV_HPP
+
+namespace cv
+{
+namespace gapi
+{
+namespace wip
+{
+namespace draw
+{
+
+// FIXME only for tests
+void GAPI_EXPORTS mosaic(cv::Mat mat, const cv::Rect &rect, int cellSz);
```

Comment:
I think we should hide those due to internal usage only. Please, remove them from the header and make static/put in namespace in src. Tests are not required as you test drawPrimitivesOCV with each primitive.